app/enterprise/1.5.x/deployment/dns-considerations.md

```diff
@@ -0,0 +1,258 @@
+
+### Cookies in the context of Kong Enterprise
+
+After you log in to Kong Manager or the Dev Portal, Kong stores session
+information in a cookie to recognize your browser during future requests. These
+cookies are created using the [session plugin][session-plugin] (via
+`admin_gui_session_conf`) or [OpenID Connect plugin][oidc-plugin].
+Configuration is more or less the same between each--the OpenID Connect plugin
+contains an embedded version of the session plugin, so while cookie handling
+code is the same, it is configured directly in the OpenID Connect plugin
+settings (`admin_gui_auth_conf`).
+
+- `cookie_name` does not affect where the cookie is used, but should be set to
+  a unique value to avoid collisions: some configurations may use the same
+  `cookie_domain` for both admin and Portal cookies, and using the same name
+  for both would then cause their cookies to collide and overwrite one another.
+- `cookie_domain` should match the common hostname suffix shared by the GUI and
+  its API. For example, if you use `api.admin.kong.example` and
+  `manager.admin.kong.example` for the Admin API and Kong Manager,
+  `cookie_domain` should be `admin.kong.example`.
+- `cookie_samesite` should typically be left at its default, `strict`. `none`
+  is not necessary if you have your DNS records and `cookie_domain` set
+  following the examples in this document. `off` is only needed if the GUI and
+  API are on entirely separate hostnames, e.g. `admin.kong.example` for the API
+  and `manager.example.com` for Kong Manager. This configuration is not
+  recommended because `off` opens a vector for cross-site request forgery
+  attacks. It may be needed in some development or testing environments, but
+  should not be used in production.
+- `cookie_secure` controls whether cookies can be sent over unsecured
+  (plaintext HTTP) requests. By default, it is set to `true`, which does not
+  permit sending the cookie over unsecured connections. This setting should
+  also remain on the default, but may be disabled in some development or
+  testing environments where HTTPS is not used.
+
+OpenID Connect uses the same settings, but prefixed with `session_`, e.g.
+`session_cookie_name` rather than `cookie_name`.
+
+Dev Portal configuration does not differ significantly from Kong Manager
+configuration, but is configured per workspace under the Portal > Settings
+section of Kong Manager, in the "Session Config (JSON)" field.
+
+As with CORS, the above is not necessary if both the GUI and API use the same
+hostname, with both behind a proxy and the API under a specific path on the
+hostname.
+
+### Troubleshooting
+
+Issues with session cookies broadly fall into cases where the cookie is not
+sent and cases where the cookie is not set. The network (see documentation for
+[Firefox][firefox-dev-network] or [Chrome][chrome-dev-network]) and
+application/storage (see documentation for [Firefox][firefox-dev-storage] or
```

Comment:
Not adding "for example" to this line since it's already clear from earlier in the sentence.